tools/package_gcsfuse_docker/Dockerfile

```diff
@@ -109,6 +109,7 @@ WORKDIR ${GCSFUSE_PKG}
 # Build the package
 RUN dpkg-deb --build ${GCSFUSE_BIN}
 RUN mv ${GCSFUSE_BIN}.deb .
+RUN rm -rf ${GCSFUSE_BIN}/DEBIAN
```

Comment:
Don't we have something like `--exclude-folder` option in fpm? `rm -rf` seems scary a bit.